{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:07:12.853Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:07:44.872Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:08:16.888Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:08:48.894Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:09:20.907Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:09:52.922Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:10:24.936Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:10:56.948Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:11:28.960Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:12:00.974Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:12:32.990Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:13:05.001Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:13:37.016Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:14:09.030Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:14:41.042Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:15:13.054Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:15:45.064Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:16:17.076Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:16:49.085Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:17:21.099Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:17:53.113Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:18:25.123Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:18:57.135Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:19:29.144Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:20:01.154Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:20:33.164Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:21:05.173Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:21:37.183Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:22:09.192Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:22:41.202Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-16T21:23:13.207Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","stack":"Error: Could not connect to MongoDB at mongodb://host.docker.internal:27017/books\n    at SafeMongooseConnection.onError (/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","service":"api","timestamp":"2023-04-16T21:29:34.622Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","stack":"Error: Could not connect to MongoDB at mongodb://host.docker.internal:27017/books\n    at SafeMongooseConnection.onError (/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","service":"api","timestamp":"2023-04-16T21:30:06.633Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","stack":"Error: Could not connect to MongoDB at mongodb://host.docker.internal:27017/books\n    at SafeMongooseConnection.onError (/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","service":"api","timestamp":"2023-04-16T21:30:38.651Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","stack":"Error: Could not connect to MongoDB at mongodb://host.docker.internal:27017/books\n    at SafeMongooseConnection.onError (/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","service":"api","timestamp":"2023-04-16T21:31:10.667Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","stack":"Error: Could not connect to MongoDB at mongodb://host.docker.internal:27017/books\n    at SafeMongooseConnection.onError (/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","service":"api","timestamp":"2023-04-16T21:31:42.680Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","stack":"Error: Could not connect to MongoDB at mongodb://host.docker.internal:27017/books\n    at SafeMongooseConnection.onError (/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://host.docker.internal:27017/books","service":"api","timestamp":"2023-04-16T21:32:14.690Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-18T15:41:51.230Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-18T15:42:23.253Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-18T15:42:55.259Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-18T15:43:27.269Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-18T15:43:59.278Z"}
{"error":{"message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","stack":"Error: Could not connect to MongoDB at mongodb://localhost:27017/shad\n    at SafeMongooseConnection.onError (/home/fabzouz/Documents/EPITECH/shad-ws/dist/src/lib/safe-mongoose-connection.js:58:31)\n    at NativeConnection.emit (node:events:513:28)\n    at /home/fabzouz/Documents/EPITECH/shad-ws/node_modules/mongoose/lib/connection.js:835:30\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"},"level":"error","message":"Could not connect to MongoDB at mongodb://localhost:27017/shad","service":"api","timestamp":"2023-04-18T15:44:31.285Z"}
